docs/development/component-checklist.md

```diff
@@ -122,7 +122,7 @@ This document provides a checklist for them that you can walk through.
 
    To ensure smooth rolling update behaviour, consider the definition of [liveness and/or readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
 
-5. **Mark node-critical components** ([example](https://github.com/gardener/gardener/blob/release-v1.64/pkg/operation/botanist/component/kubeproxy/resources.go#L325))
+5. **Mark node-critical components** ([example](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/component/kubeproxy/resources.go#L325-L328))
```

Comment:
I would suggest not using the master branch as the code at these lines can change. Rather use a commit hash.
```suggestion
5. **Mark node-critical components** ([example](https://github.com/gardener/gardener/blob/ec6f410d180868fded06122f53be16ed9892353f/pkg/operation/botanist/component/kubeproxy/resources.go#L325-L328))
```